docs/source/features.rst

```diff
@@ -20,6 +49,25 @@ Limitations
 Dependency detection
 --------------------
 Whilst fab can automatically determine dependencies from module use statements,
-and from standalone call statements, it doesn't currently detect a dependency from a call statement
-on a single-line if statement. We can pass the analyser any dependencies which Fab can't detect,
-and they'll make their way through to the link stage.
\ No newline at end of file
+and from standalone call statements, it doesn't currently detect a dependency from a call statement on a
+single-line if statement: `IF (x .GT. 0) CALL foo()`.
+
+We can pass the analyser any symbol dependencies which Fab can't detect.
+The files which contain them, *and their dependencies*, will make their way through to the compile and link stages.
+This is done using the `unreferenced_deps` argument to the :class:`~fab.steps.analyse.Analyse` step.
```

Comment:
Would a code snippet showing how to do this be too big?

Reply:
No that's a good idea.